manifests/n/NXLog/NXLog-CE/2.10.2150/NXLog.NXLog-CE.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: NXLog.NXLog-CE
+PackageVersion: 2.10.2150
+PackageLocale: en-US
+Publisher: NXLog Ltd
+PackageName: NXLog Community Edition

```

Comment:
```suggestion
PackageName: NXLog-CE
```

Reply:
This change so that winget can upgrade the application when someone uses the `winget upgrade` command.